packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -895,6 +957,15 @@ export function getResponseMapping(
   return props;
 }
 
+function reshapeValue(type: Type) {
+  switch (type.type) {
+    case "datetime":
+      return "(value) => new Date(value as string)";
+    default:
```

Comment:
missing bytes ? and datetime serialization is another case

Reply:
yes, I'll add both of them